framework/wazuh/core/config/models/comms_api.py

```diff
@@ -0,0 +1,66 @@
+from pydantic import BaseModel, PositiveInt, PositiveFloat, FilePath
```

Comment:
We are using pydantic for the Comms API validations only, we should discuss whether to use it in other parts of the code

Reply:
Added in the other classes in e4e54327c2b3a45cbe77a6f9c30cfd19dc9ce0b6.